TranslationApp/TranslationHelpers.cs

```diff
@@ -19,12 +19,12 @@ internal static class TranslationHelpers
                 GitCommands.AppSettings.CurrentTranslation = "";
 
                 var translatableTypes = TranslationUtl.GetTranslatableTypes();
-                foreach (var types in translatableTypes)
+                foreach (var (key, types) in translatableTypes)
```

Comment:
In this file I left quite a few `key` symbols because I wasn't confident what they should be called. I think they may be `language`.